terragraph_planner/data/template.yaml

```diff
@@ -112,189 +110,150 @@ LINE_OF_SIGHT:
       NODE_CAPEX: 250.0
 
-      # [Int] Maximum number of radio nodes allowed on each site.
-      # For CNs, this input will be ignored as it is always 1.
-
+      # [Int] Maximum number of radio nodes allowed on each site. For CNs, this
+      # input must be 1.
       NUMBER_OF_NODES_PER_SITE: 4
 
       # The set of radio specification parameters.
-
       SECTOR_PARAMS:
-        # [Float] Per-sector horizontal beamforming scan range of
-        # the antenna in degrees.
-
+        # [Float] Per-sector horizontal beamforming scan range of the antenna
+        # in degrees.
         HORIZONTAL_SCAN_RANGE: 70.0
 
         # [Int] Number of sectors in each node.
-
         NUMBER_SECTORS_PER_NODE: 1
 
-        # [Float] Antenna gain at boresight (dBi).
-
+        # [Float] Antenna gain at the boresight (dBi).
         ANTENNA_BORESIGHT_GAIN: 30.0
 
-        # [Float] Maximum Transmission power in dBm.
-
+        # [Float] Maximum transmission power in dBm.
         MAXIMUM_TX_POWER: 16.0
 
-        # [Optional][Float] Minimum Transmission power in dBm.
-
+        # [Optional][Float] Minimum transmission power in dBm.
         MINIMUM_TX_POWER: null
 
-        # [Float] Transmitter diversity gain in dB
-        # (e.g., polarization diversity).
-
+        # [Float] Transmitter and receiver diversity gain in dB (e.g.,
+        # polarization diversity).
         TX_DIVERSITY_GAIN: 0.0
-
-        # [Float] Receiver diversity gain in dB
-        # (e.g., polarization diversity).
-
         RX_DIVERSITY_GAIN: 0.0
 
-        # [Float] Miscellaneous losses on the transmitter
-        # in dB (e.g., cable losses).
-
-        TX_MISCELLANEOUS_LOSS: 0.0
-
-        # [Float] Miscellaneous losses on the receiver in dB
+        # [Float] Miscellaneous losses on the transmitter and receiver in dB
         # (e.g., cable losses).
-
+        TX_MISCELLANEOUS_LOSS: 0.0
         RX_MISCELLANEOUS_LOSS: 0.0
 
         # [Optional][Float] The minimum MCS level allowed.
-
         MINIMUM_MCS_LEVEL: null
 
-        # [String] Antenna pattern file defining the signal loss
-        # of the antenna in different angles in Planet's format (txt).
-
+        # [String] Antenna pattern file defining the signal loss of the antenna
+        # at different angles in Planet's format. File Type: TXT
         ANTENNA_PATTERN_FILE_PATH: antenna_pattern.txt
 
-        # [String] Scan pattern file defining the signal gain of
-        # the antenna boresight in different scan angles (csv).
-
-        SCAN_PATTERN_FILE_PATH: scan_patter.csv
-
-        # [String] Scan pattern file contains the mapping between MCS, SNR, Mbps
-        # and TX backoff (csv).
+        # [String] Scan pattern file defining the signal gain of the antenna
+        # boresight at different scan angles. File type: CSV
+        SCAN_PATTERN_FILE_PATH: scan_pattern.csv
 
+        # [String] MCS table file containing the mapping between MCS, SNR,
+        # throughput (in Mbps), and Tx backoff. File type: CSV
```

Comment:
Maybe add units for Tx backoff if that makes sense. (Don't need for SNR unless it's something other than dB)